website/src/schema-markup/sdk-reference/android.json

```diff
@@ -0,0 +1,58 @@
+{
+  "@context": "https://schema.org/", 
+  "@type": "HowTo", 
+  "name": "How to use feature flags in Android Java?",
+  "description": "How to use feature flags in Android (Java) using ConfigCat Feature Flags.",
+  "image": "https://configcat.com/images/shared/home.png",
+  "totalTime": "PT10M",
+  "estimatedCost": {
+    "@type": "MonetaryAmount",
+    "currency": "USD",
+    "value": "0"
+  },
+  "supply": {
+    "@type": "HowToSupply",
+    "name": "ConfigCat"
```

Comment:
I'm not sure if it is needed here, but in the net.json i see image and url properties here too.

Reply:
hmmm the image and url properties were not generated by the [schema markup genrator tool](https://technicalseo.com/tools/schema-markup-generator/) but looking [here](https://schema.org/HowToSupply) i can see that it is indeed required.